oio/cli/admin/cluster.py

```diff
@@ -157,9 +167,66 @@ def _unlock_all(self):
                 except Exception as exc:
                     yield type_, descr['addr'], str(exc)
 
-    def take_action(self, _parsed_args):
+    def take_action(self, parsed_args):
         columns = ('Type', 'Service', 'Result')
-        return columns, self._unlock_all()
+        return columns, self._unlock_all(parsed_args)
+
+
+class ClusterWait(lister.Lister):
+    """Wait for the services to get a score"""
+
+    log = logging.getLogger(__name__ + '.ClusterWait')
+
+    def get_parser(self, prog_name):
+        parser = super(ClusterWait, self).get_parser(prog_name)
+        parser.add_argument(
+            '-t', '--type',
```

Comment:
I would prefer positional arguments, as in `ClusterList` and `ClusterLocalList`.